ui/src/app/components/side-nav.tsx

```diff
@@ -316,7 +317,8 @@ export class SideNav extends React.Component<SideNavProps, SideNavState> {
       />
       {
         (profile.authorities.includes(Authority.ACCESSCONTROLADMIN)
-          || profile.authorities.includes(Authority.COMMUNICATIONSADMIN)) && <SideNavItem
+          || profile.authorities.includes(Authority.COMMUNICATIONSADMIN)
+          || profile.authorities.includes(Authority.WORKSPACESVIEW)) && <SideNavItem
```

Comment:
Wow – does Swagger-codegen really just collapse tokens like this? Bummer.

Reply:
I'm guessing `WORKSPACES_VIEW` -> `WORKSPACESVIEW`.  I was quite surprised to see that the first time I did.  But it looks like all the UI does this.